net-im/nheko/nheko-0.11.3.recipe

```diff
@@ -113,4 +119,6 @@ INSTALL()
 
 	addResourcesToBinaries nheko.rdef $appsDir/Nheko
 	addAppDeskbarSymlink $appsDir/Nheko
+
+	prepareInstalledDevelLib libmatrix_client
```

Comment:
What's the benefit? The static libs shouldn't be needed at runtime.

Reply:
It comes included as bundled mtxclient to fix the build, should be fine to remove on a version bump for Nheko

Reply:
I could run another check to see if it launches fine building with the bundled one and using our shared library in the mtxclient package.

Reply:
I mean, the headers and the libs shouldn't be in the mheko package. Please remove them.

Reply:
Was looking into that when checking earlier, should be resolved in https://github.com/haikuports/haikuports/pull/9776